_src/pl/static/js/checkbox-replace.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Checkbox replacement
+ * Handles functionality for the replacement checkboxes, which
+ * allows us to style them to our liking. Accessibility is main-
+ * tained via use of the default checkbox control. The 
+ * replacement control is not visible to screen readers.
+ */
+
+$(function() {
+
+    var CustomCheckboxReplacement = {
+
+        vars: {
+            replaced:       $('.replace-checkbox'),
+            replacedClass:  'is-replaced is-transparent',
+            customClass:    'wrapper-custom-checkbox',
+            wrapperClass:   'wrapper-replace-checkbox',
+            iconChecked:    'icon-check-square',
+            iconUnchecked:  'icon-square-o'
+        },
+
+        init: function() {
+            this.replaceFoundCheckboxs();
+            this.listenForCheckboxClick();
+        },
+
+        replaceFoundCheckboxs: function() {
+            var that = this;
+
+            if (that.vars.replaced.length) {
+
+                that.vars.replaced.each(function() {
+                    $(this).addClass(that.vars.replacedClass);
+                    $(this).wrap('<div class="' + that.vars.wrapperClass + '"></div>');
+                    $(this).parent().append($('<div class="' + that.vars.customClass + '" aria-hidden="true"></div>'));
+                });
+
+                that.setCheckboxsSelected();
+            }
+        },
+
+        listenForCheckboxClick: function() {
+            var that = this;
```

Comment:
Do not need this here. If this doesn't work within the click handler (sometimes it will) please pass the function you are using to the .on handler function (see example http://jsfiddle.net/AlasdairSwan/5kt6vLt3/)